stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -1,13 +1,18 @@
 [id="proc-aap-gcp-backup-and-restore"]
 
```

Comment:
Since we separated, restore in another file. we can change here `proc-aap-gcp-backup-process` and also the file name as `proc-aap-gcp-backup-process.adoc`